keyboards/eniigmakeyboards/ek87/ek87.h

```diff
@@ -0,0 +1,203 @@
+/* Copyright 2020 adamws
```

Comment:
## General thought on this file

I'm having the sensation that the the Split Backspace and Split Shift layout macros are superfluous and could be deleted. You could probably go with just the `_all`, `_ansi`, `_ansi_7uspace`, `_iso` and `_iso_7uspace` layouts and that'd be enough for the overwhelming majority of users, and this doesn't prevent the available layouts from being extended later.

Also, IMO some of the layout macros should be renamed, to enable support for the [Community Layouts feature](https://docs.qmk.fm/#/feature_layouts).

- `LAYOUT_ansi` -> `LAYOUT_tkl_ansi`
- `LAYOUT_ansi_7uspace` -> `LAYOUT_tkl_ansi_tsangan`
- `LAYOUT_iso` -> `LAYOUT_tkl_iso`
- `LAYOUT_iso_7uspace` -> `LAYOUT_tkl_iso_tsangan`

These renames should be made throughout your keyboard's codebase.

Reply:
Fixed this